### ISSUE TITLE
realtime_panoptic/models/backbones.py", line 65, in __init__     extra_blocks=LastLevelP6P7(out_channels, out_channels),   File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/feature_pyramid_network.py", line 53, in __init__     raise ValueError("in_channels=0 is currently not supported") ValueError: in_channels=0 is currently not supported

### ISSUE THREAD

**Issue by jinfagang** [TRI-ML/realtime_panoptic#4]
```
realtime_panoptic/models/backbones.py", line 65, in __init__
    extra_blocks=LastLevelP6P7(out_channels, out_channels),
  File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/feature_pyramid_network.py", line 53, in __init__
    raise ValueError("in_channels=0 is currently not supported")
ValueError: in_channels=0 is currently not supported

```

**Comment by jieli352** [comment]
Would you provide more details of your run? E.g. config file and system environment? Thank you!

**Comment by jinfagang** [comment]
@jieli352  this error happens with torch 1.6 (I believe also 1.5) and root reason is accordingly torchvision which FPN used inside it.

The in_channel can not be 0 in some updates in torchvision, I don't know how to edit it. I am using default code without any modification.

**Comment by jieli352** [comment]
Thanks for the information. The update in PyTorch here [PR-1407](https://github.com/pytorch/vision/pull/1407) now requires filtered ResNet layers as FPN inputs. I have incorporated this change into our recent PR: #5 . That should unblock you from using PyTorch 1.6.

**Comment by pushkar-khetrapal** [comment]
hi @jinfagang I was facing the same error and I resolved it using downgrading the torch and torchvision versions.

`pip install future typing numpy awscli
pip install https://download.pytorch.org/whl/cu100/torch-1.1.0-cp36-cp36m-linux_x86_64.whl
pip install https://download.pytorch.org/whl/cu100/torchvision-0.3.0-cp36-cp36m-linux_x86_64.whl`

run these commands then code starts working.

**Comment by AdrienGaidon-TRI** [comment]
Hi! We just merged #5 so please let us know if you still have the issue.
